content/en/docs/tasks/administer-cluster/access-cluster-api.md

```diff
@@ -151,16 +155,23 @@ describes how you can configure this as a cluster administrator.
 
 ### Programmatic access to the API
 
-Kubernetes officially supports client libraries for [Go](#go-client), [Python](#python-client), [Java](#java-client), [dotnet](#dotnet-client), [JavaScript](#javascript-client), and [Haskell](#haskell-client). There are other client libraries that are provided and maintained by their authors, not the Kubernetes team. See [client libraries](/docs/reference/using-api/client-libraries/) for accessing the API from other languages and how they authenticate.
+Kubernetes officially supports client libraries for [Go](#go-client), [Python](#python-client),
+[Java](#java-client), [dotnet](#dotnet-client), [JavaScript](#javascript-client), and
+[Haskell](#haskell-client). There are other client libraries that are provided and maintained by
+their authors, not the Kubernetes team. See [client libraries](/docs/reference/using-api/client-libraries/)
+for accessing the API from other languages and how they authenticate.
 
 #### Go client
 
-* To get the library, run the following command: `go get k8s.io/client-go@kubernetes-<kubernetes-version-number>` See [https://github.com/kubernetes/client-go/releases](https://github.com/kubernetes/client-go/releases) to see which versions are supported.
+* To get the library, run the following command: `go get k8s.io/client-go@kubernetes-<kubernetes-version-number>`
+  See [https://github.com/kubernetes/client-go/releases](https://github.com/kubernetes/client-go/releases)
+  to see which versions are supported.
 * Write an application atop of the client-go clients.
 
 {{< note >}}
 
-client-go defines its own API objects, so if needed, import API definitions from client-go rather than from the main repository. For example, `import "k8s.io/client-go/kubernetes"` is correct.
+client-go defines its own API objects, so if needed, import API definitions from client-go rather than
```

Comment:
nit: `client-go` is a library. Does it make sense to surround the name of the library in code blocks?